src/components/Header.js

```diff
@@ -21,6 +20,16 @@ const Header = styled.header`
   }
 `;
 
+const Logo = css`
```

Comment:
Qu'est-ce qui diffère entre styled components et émotions pour qu'on aie besoin des 2?

On applique le `@média` de la même manière. Est-ce que `styled.span` wrap le component dans un `span`?

On aurait pu utiliser `fill=currentColor` sur le SVG?

Reply:
Hmm good idea! I’ll try 🙂

Reply:
It worked! 🎉 👍